package ru.otus.task4.mathtest.exc;

public class QuestionsLoadException extends Exception {
    public QuestionsLoadException(Throwable cause) {
        super(cause);
    }
}
